src/blueprint/server.rs

```diff
@@ -108,29 +105,18 @@ fn handle_response_headers(resp_headers: BTreeMap<String, String>) -> Valid<Head
   Ok(response_headers)
 }
 
-fn handle_base_url(base_url: Option<String>) -> Valid<Option<String>, String> {
-  let base_url = if let Some(base_url) = base_url {
-    Valid::Ok(reqwest::Url::parse(base_url.as_str()).map_err(|e| ValidationError::new(e.to_string()))?)?;
-    Some(base_url)
-  } else {
-    None
-  };
-  Ok(base_url)
-}
```

Comment:
We have deleted this validation I think. Can you add it back with a unit test?

Reply:
oops. okay, i will add it back and invoke it in the config_blueprint  process. according to unit test, do you mean for handle_base_url function or handle_base_url(the function call handle_base_url)

Reply:
here 
https://github.com/tailcallhq/tailcall/pull/479/files/7869161c5912cf7b0d6fe32f77841e58922084b5#diff-0934da3dca97565bf71c4fd107919b495a6a971c292009709dfc4756760ac27eR35-R51